tests/unit/services.py

```diff
@@ -0,0 +1,180 @@
+from unittest import mock
+
+import pytest
+
+from lms.models import ApplicationSettings
+from lms.services.application_instance import ApplicationInstanceService
+from lms.services.assignment import AssignmentService
+from lms.services.blackboard_api.service import BlackboardAPIClient
+from lms.services.canvas_api import CanvasAPIClient
+from lms.services.course import CourseService
+from lms.services.grading_info import GradingInfoService
+from lms.services.grant_token import GrantTokenService
+from lms.services.group_info import GroupInfoService
+from lms.services.grouping import GroupingService
+from lms.services.h_api import HAPI
+from lms.services.http import HTTPService
+from lms.services.launch_verifier import LaunchVerifier
+from lms.services.lti_h import LTIHService
+from lms.services.lti_outcomes import LTIOutcomesClient
+from lms.services.oauth1 import OAuth1Service
+from lms.services.oauth2_token import OAuth2TokenService
+from lms.services.vitalsource import VitalSourceService
+from tests import factories
+
+__all__ = (
+    # Meta fixture for creating service fixtures
+    "mock_service",
+    # Individual services
+    "application_instance_service",
+    "assignment_service",
+    "blackboard_api_client",
+    "canvas_api_client",
+    "course_service",
+    "grading_info_service",
+    "grant_token_service",
+    "group_info_service",
+    "grouping_service",
+    "http_service",
+    "launch_verifier",
+    "lti_h_service",
+    "lti_outcomes_client",
+    "oauth1_service",
+    "oauth2_token_service",
+    "h_api",
+    "vitalsource_service",
```

Comment:
We could do without this, but it means `conftest.py` ends up re-importing things like `factories` and `pytest` and using them from here and not importing it's own. Which is a bit weird.